package.json

```diff
@@ -30,13 +30,15 @@
     "react-bootstrap": "^1.1.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "redux-form": "^8.3.6"
```

Comment:
What is `redux-form` being used for?

As a side note I'd recommend `formik` but plain React forms work

Reply:
OK. I used some labels told me to install redux-form, but I don't think I use it.  I can delete this.